mlflow/models/evaluation/default_evaluator.py

```diff
@@ -522,6 +522,18 @@ def _get_aggregate_metrics_values(metrics):
 }
 
 
+def _extract_chat_message_response_or_prediction(prediction: list) -> str:
+    # Handle case when ChatMessage sequence is returned by the predict func
+    if (
+        prediction is not None
+        and isinstance(prediction, list)
+        and all(isinstance(_p, dict) for _p in prediction)
```

Comment:
```suggestion
        and all(isinstance(message, dict) for message in prediction)
```
No need to do shorthand notation like that for comprehensions.